persistence/src/main/java/com/epam/rft/atsy/persistence/entities/StateEntity.java

```diff
@@ -6,50 +6,35 @@
 import java.util.Date;
 
 
-@Data
-@NoArgsConstructor
-@AllArgsConstructor
-@EqualsAndHashCode(callSuper = true)
-@ToString(callSuper = true)
-@Entity
-@Table(name = "states", schema = "atsy")
+@Data @NoArgsConstructor @AllArgsConstructor @EqualsAndHashCode(callSuper = true)
```

Comment:
As per my understanding the google style prohibits this formatting, see https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations